src/imitation/data/wrappers.py

```diff
@@ -206,3 +211,124 @@ def step(self, action):
                 "rews": np.stack(self._rews),
             }
         return obs, rew, terminated, truncated, info
+
+
+class HumanReadableWrapper(gym.Wrapper):
```

Comment:
May be able to simplify it by subclassing the more specialized [ObservationWrapper](https://github.com/Farama-Foundation/Gymnasium/blob/aabf65cdc31431cb0b3737f8afb27d4af0fc50e7/gymnasium/core.py#L480) since it seems this wrapper only changes the observation (and not action/reward).